src/api/public/apidocs-new/paths/my_notifications.yaml

```
@@ -6,14 +6,17 @@ get:
   parameters:
     - in: path
       name: project
-      required: false
       schema:
         type: string
     - in: path
       name: page
-      required: false
       schema:
         type: int
+    - in: path
```

```suggestion
    - in: query
```

I'm going to open another PR to change the other parameters too and add a few more that are missing...